### PR TITLE
Fix Table examples #trivial

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Update `Table` examples.

#### What problem is this solving?
Portuguese i18n.

#### How should this be manually tested?
`yarn styleguide`

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Table/README.md [hunk 1]
```diff
@@ -56,17 +56,17 @@ const defaultSchema = {
 The Schema property is a JSON used to define the table columns and how they should behave visually. The Schema has properties and each one of them defines a column in the table.
 Example with simple structure:
 
-```md
+```python
```
Comment (estacioneto) [inline react/components/Table/README.md:59]: Why not `js`?

Comment (kevinch) [reply react/components/Table/README.md:59]: It's a hack to display the code. This is not living code.
Screen Shot 2020-05-08 at 17 09 24

Comment (estacioneto) [reply react/components/Table/README.md:59]: Maybe adding `static` on it?

https://github.com/styleguidist/react-styleguidist/blob/master/docs/Documenting.md#usage-examples-and-readme-files

Comment (kevinch) [reply react/components/Table/README.md:59]: Good one. Less hacky, i did not know about that one.

**Pull request merged**